doc/configuration.md

```diff
@@ -51,6 +51,13 @@ Reference
 
 ### Core
 
+#### php.version
```

Comment:
=> `code.php_version` in general all config keys should be prefixed with their module name.